asset_stream_manager/asset_stream_config.cc

```diff
@@ -106,31 +161,31 @@ absl::Status AssetStreamConfig::LoadFromFile(const std::string& path) {
                         reader.getFormattedErrorMessages()));
   }
 
-#define ASSIGN_VAR(var, flag, type)        \
-  do {                                     \
-    if (config.isMember(#flag)) {          \
-      var = config[#flag].as##type();      \
-      flags_read_from_file_.insert(#flag); \
-    }                                      \
+#define ASSIGN_VAR(var, flag, type)       \
+  do {                                    \
+    if (config.isMember(flag)) {          \
```

Comment:
I am not so good with macros. Why is # not required anymore for reading the flag?

Reply:
# makes a string out of a non-string, e.g. the macro FOO(v) #v makes "bar" out of Foo(bar).
Since I switched from _ to - (so it matches cdc_rsync's style), clang-format was freaking out for things like
  ASSIGN_VAR(session_cfg_.fuse_cleanup_timeout_sec, cleanup-timeout, Int);
(without the quotes). It added spaces around the -. Thus, I quoted the strings and removed the # from the macro.